src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -343,6 +343,13 @@ Copyright (c) .NET Foundation. All rights reserved.
     
   </Target>
 
+  <Target Name="_CheckForFailedSDKResolution"
+        BeforeTargets="_CheckForInvalidConfigurationAndPlatform"
```

Comment:
Does this target run during design-time build?  I think not, but can you make sure we don't prevent resolving framework references and NuGet packages so there's a decent intellisense experience.

Reply:
I don't have a good understanding of design time builds, so I don't know. Right now this target runs before resolving framework references and packages in the build, does that interfere with intellisense? Should I hook it in later in the build?

Reply:
There are targets which gather the references to pass to the compiler for intellisense and other tooling.  We don't want to prevent those targets from running successfully.  

I think that this target probably isn't run for the design time build, but check to make sure that when the global.json resolution fails, that in VS you can add a PackageReference and get Intellisense for APIs from the package as well as from .NET Core.

If the target is run, then we can avoid the check in design time builds, something like this: https://github.com/dotnet/sdk/blob/9b6505ef20113424d1f694463afa799438451a1b/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets#L312-L316

Reply:
I see, I did some manual testing and I was able to add a package and get intellisense for the package APIs, so we should be good here.